provider/kubernetes.go

```diff
@@ -27,6 +27,9 @@ const (
 	ruleTypePathStrip          = "PathStrip"
 	ruleTypePath               = "Path"
 	ruleTypePathPrefix         = "PathPrefix"
+
+	annotationKubernetesWhitelistSourceRange = "ingress.kubernetes.io/whitelist-source-range"
+	annotationTraeffikWhitelistSourceRange   = "traefik.frontend.whitelistSourceRange"
```

Comment:
Do we actually have to support this alternative format? Can't we just support the more Kube-idiomatic annotation above?